src/main/java/Echo.java

```diff
@@ -0,0 +1,131 @@
+import parser.Parser;
+import tasks.Task;
+import storage.Storage;
+import tasks.TaskList;
+import ui.Ui;
+
+import java.io.IOException;
+import java.time.format.DateTimeParseException;
+
+public class Echo {
+    private String word;
+    private TaskList list;
+    private final Storage storage;
+    private final Ui ui;
+    private final Parser parser;
+
+    public Echo() {
+        this.storage = new Storage("data/chatHistory.txt");
+        try {
+            this.list = storage.load();
+        } catch (IOException e) {
+            this.list = new TaskList();
+        }
+        this.ui = new Ui();
+        this.parser = new Parser();
+    }
+
+    /**
+     * Sets the word for the echo object.
+     * @param word used to initialise the word of an echo object.
+     */
+    public void setWord(String word) {
+        this.word = word;
+    }
+
+    /**
+     * Main logic for the entire chatbot. Handles all the different commands.
+     */
+    public String echoOut() {
+
+            String description = word;
+            String[] parts = description.split(" ", 2);
```

Comment:
Not sure that line 41 is needed. If "description" is only used in this instance, is it safe to use "word" directly?